crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -57,5 +58,9 @@ def compute_resources_available?
       # FIXME: to be implemented
       true
     end
+
+    def ceph_healthy?
```

Comment:
It will be rebased after the PR with other "healthy" method is merded.

I'm not sure about `storages`, this is really specific to ceph and we do not have any other kind of check